src/api/resources/sample_units_base.py

```diff
@@ -21,6 +21,8 @@ class SampleUnitSerializer(BaseAPISerializer):
         max_digits=3,
         decimal_places=1,
         coerce_to_string=False,
+        min_value=0,
```

Comment:
coderabbit is right -- we should change src/api/submission/protocol_validations.py to be 0,40.
And here as elsewhere, remove each model field validator for which we're adding a serializer field validator.